### PR TITLE
chore: update GitHub templates

### PR DESCRIPTION
**Pull request by kazo0**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/success-story.md [hunk 1]
```diff
@@ -1,16 +1,15 @@
...
 labels: kind/consumer-experience, kind/documentation, triage/untriaged
 ---
 
-<!-- STOP -- PLEASE READ!
+<!--
 
-We 💖 to hear about how, where and what you're doing with Uno. Sharing this information is one of the kindest things you can do in open-source and the maintainers love hearing about success stories. Instead of raising a new GitHub issue could you please comment in the existing thread?
-
-https://github.com/nventive/Uno/issues/18#issuecomment-494887105
+We 💖 to hear about how, where and what you're doing with the Uno Toolkit. Sharing this information is one of the kindest things you can do in open-source and the maintainers love hearing about success stories.
```
Comment (agneszitte-nventive) [inline .github/ISSUE_TEMPLATE/success-story.md:9]:
```suggestion
We 💖 to hear about how, where, and what you're doing with the Uno Toolkit. Sharing this information is one of the kindest things you can do in open-source and the maintainers love hearing about success stories.
```

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -22,28 +22,23 @@ What kind of change does this PR introduce?
...
 ## PR Checklist
 
 Please check if your PR fulfills the following requirements:
 
 - [ ] Tested code with current [supported SDKs](../README.md#supported)
-- [ ] Docs have been added/updated which fit [documentation template](https://github.com/nventive/Uno/blob/master/doc/.feature-template.md). (for bug fixes / features)
-- [ ] [Unit Tests and/or UI Tests](doc/articles/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
-- [ ] [Wasm UI Tests](doc/articles/working-with-the-samples-apps.md#running-the-webassembly-ui-tests-snapshots) are not showing unexpected any differences. Validate PR `Screenshots Compare Test Run` results.
+- [ ] [Docs](https://github.com/unoplatform/uno.toolkit.ui/tree/main/doc) have been added/updated
+- [ ] Runtime Tests and/or UI Tests for the changes have been added (for bug fixes / features) (if applicable)
```
Comment (agneszitte-nventive) [inline .github/pull_request_template.md:35]: Documentation links for the tests can be kept I think. Just use the proper link to the uno repo or unoplatform doc link no ?

**Review by Xiaoy312 — APPROVED** [review]

File: .github/config.yml [hunk 3]
```diff
@@ -16,7 +16,7 @@
 
 # Comment to be posted to on pull requests merged by a first time user
 firstPRMergeComment: >
-  Congrats on your first pull-request! We ❤ the people who are involved in this project, and we’d love to pay it forward by [sending you some swag](https://forms.office.com/Pages/ResponsePage.aspx?id=Ye9TbdG2UEGuC0O5DnXgzcxnnVo6zspHjmNPgM3UO41UNFpFU01TV1UxU0FWRTNMNFJUTlBLMjhIWi4u). If you have any feedback (or ideas how to improve Uno as a open-source project) please [open a feedback issue](https://github.com/unoplatform/uno/issues/new?labels=kind%2Ffeedback%2C+triage%2Funtriaged&template=feedback.md).
+  Congrats on your first pull-request! We ❤ the people who are involved in this project, and we’d love to pay it forward by [sending you some swag](https://forms.office.com/Pages/ResponsePage.aspx?id=Ye9TbdG2UEGuC0O5DnXgzcxnnVo6zspHjmNPgM3UO41UNFpFU01TV1UxU0FWRTNMNFJUTlBLMjhIWi4u). If you have any feedback (or ideas how to improve the Uno Toolkit as an open-source project) please [open a feedback issue](https://github.com/unoplatform/uno.toolkit.ui/issues/new?assignees=&labels=kind%2Ffeedback%2C+triage%2Funtriaged&template=feedback.md).
```
Comment (Xiaoy312) [inline .github/config.yml:19]:
```suggestion
  Congrats on your first pull-request! We ❤ the people who are involved in this project, and we'd love to pay it forward by [sending you some swag](https://forms.office.com/Pages/ResponsePage.aspx?id=Ye9TbdG2UEGuC0O5DnXgzcxnnVo6zspHjmNPgM3UO41UNFpFU01TV1UxU0FWRTNMNFJUTlBLMjhIWi4u). If you have any feedback (or ideas on how to improve the Uno Toolkit as an open-source project) please [open a feedback issue](https://github.com/unoplatform/uno.toolkit.ui/issues/new?assignees=&labels=kind%2Ffeedback%2C+triage%2Funtriaged&template=feedback.md).
```

File: .github/pull_request_template.md [hunk 4]
```diff
@@ -22,28 +22,23 @@ What kind of change does this PR introduce?
...
 - [ ] Contains **NO** breaking changes
-- [ ] Updated the [Release Notes](https://github.com/nventive/Uno/tree/master/doc/ReleaseNotes)
 - [ ] Associated with an issue (GitHub or internal)
```
Comment (Xiaoy312) [inline .github/pull_request_template.md:37]:
```suggestion
- [ ] Associated with an issue (GitHub or internal)
- [ ] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.
```

**Pull request merged**